upgrading.md

```diff
@@ -0,0 +1,36 @@
+Upgrading OpenSearch Python Client
+==================================
+
+### Upgrading to >= 2.2.0
+
+Opensearch-py now includes opensearch-dsl-py features. Opensearch-dsl-py was merged into opensearch-py in the following PR [#287](https://github.com/opensearch-project/opensearch-py/pull/287). 
```

Comment:
* Spell as `opensearch-py` and `opensearch-dsl-py`.
* Add the pypi link for opensearch-dsl-py.
* You can also say "opensearch-dsl-py` was merged into `opensearch-py` preserving backwards compatibility with the previous `opensearch-py` version. (Refer link)"